corehq/apps/hqadmin/templates/hqadmin/email/superuser_staff_email.html

```diff
@@ -0,0 +1,34 @@
+<p>The following users had changes to their superuser and staff statuses:</p>
+
+<table>
+  <tbody>
+    <tr>
+      <td><u>Users</u></td>
+      <td><u>is Staff</u></td>
+      <td><u>is Superuser</u></td>
+    </tr>
+    {% for user in user_changes %}
+      <tr>
+        <td>{{ user.email }}</td>
+        {% if 'is_staff' in user %}
+          <td><s>{{ user.is_staff|yesno:"False,True" }}</s> -> <b>{{ user.is_staff }}</b></td>
+        {% elif 'same_staff' in user %}
+          <td>{{ user.same_staff }}</td>
+        {% endif %}
+        {% if 'is_superuser' in user %}
+          <td><s>{{ user.is_superuser|yesno:"False,True" }}</s> -> <b>{{ user.is_superuser }}</b></td>
+        {% elif 'same_superuser' in user %}
+          <td>{{ user.same_superuser }}</td>
+        {% endif %}
+      </tr>
+    {% endfor %}
+    </tbody>
+</table>
+
+<p>You've been sent this email because you are a Django Admin.</p>
```

Comment:
I think "Django Admin" isn't the right term here. I think you can leave out this message entirely. It's hard to describe in a lay way (practically the email(s) in `settings.ADMINS` will be shared email lists not individual people), and developers who want to know who gets the email can easily look it up by searching the codebase.